app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,71 @@
+<footer class="page-footer font-small indigo">
+  <div class="container text-center text-md-left">
+    <div class="row">
+      <div class="col-md-3 mx-auto">
+      <h5 class="font-weight-bold text-uppercase mt-2 mb-2">Connect</h5>
+
+      <ul class="list-unstyled">
+      <li>
+      <a href="#!">Opportunities</a>
+      </li>
+      <li>
+      <a href="#!">Slack</a>
+      </li>
+      </div>
+
+      <hr class="clearfix w-100 d-md-none">
+
+
+        <div class="col-md-3 mx-auto">
+
+
+          <h5 class="font-weight-bold text-uppercase mt-2 mb-2">Inspire</h5>
+
+          <ul class="list-unstyled">
+            <li>
+              <a href="#!">Getting Started</a>
+            </li>
+            <li>
+              <a href="#!">Learn Ruby On Rails</a>
+            </li>
+            <li>
+              <a href="#!">Wiki</a>
+            </li>
+          </ul>
+
+        </div>
+
+        <hr class="clearfix w-100 d-md-none">
+
+
+        <div class="col-md-3 mx-auto">
+
+
+          <h5 class="font-weight-bold text-uppercase mt-2 mb-2">Share</h5>
+
+          <ul class="list-unstyled">
+            <li>
+              <a href="#!">News</a>
+            </li>
+            <li>
+              <a href="#!">Events</a>
+            </li>
+            <li>
+              <a href="#!">Our Story</a>
+            </li>
+          </ul>
+
+        </div>
+        </div>
+        </div>
+        </footer>
+
+    <!-- Copyright -->
+    <div class="footer-copyright text-center py-3">© 2019 Copyright:
+      <a href="http://yegrb.com/index.html/"> YEGRB.com</a>
```

Comment:
This should be root_path. Never hard code links to your own website :smile:

Reply:
Haha, ill be fixing all the links on my next iteration 😂

Reply:
:+1: